test_windows.js

```diff
@@ -0,0 +1,19 @@
+const spawn = require('child_process').spawn;
```

Comment:
I think it would be better to put this in the `bin` directory with the other scripts used for testing.

Also, for completeness we should add a license to the top of the file.

Reply:
Done.